src/support/azext_support/aaz/latest/support/in_subscription/file/_show.py

```diff
@@ -175,17 +175,17 @@ def _build_schema_on_200(cls):
             )
 
             properties = cls._schema_on_200.properties
-            properties.chunk_size = AAZFloatType(
+            properties.chunk_size = AAZIntType(
                 serialized_name="chunkSize",
             )
             properties.created_on = AAZStrType(
                 serialized_name="createdOn",
                 flags={"read_only": True},
             )
-            properties.file_size = AAZFloatType(
+            properties.file_size = AAZIntType(
```

Comment:
Why has the type changed

Reply:
Because file size can only be int, this is correct

Reply:
So why was it incorrect before? Does this mean we need changes in custom files also?

Reply:
This is one of the changes Vijaya made to the sdk. I was manually making it int ( and it is correct in custom and we have cast to ensure int on top of it)